library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -627,7 +627,12 @@ libspdm_return_t libspdm_build_response(void *spdm_context, const uint32_t *sess
                        libspdm_get_scratch_buffer_large_message_offset(spdm_context);
         large_buffer_size = libspdm_get_scratch_buffer_large_message_capacity(spdm_context);
 
-        if (my_response_size < large_buffer_size) {
+        if (my_response_size > context->connection_info.capability.max_spdm_msg_size) {
+            status = libspdm_generate_error_response(context,
+                                                     SPDM_ERROR_CODE_RESPONSE_TOO_LARGE,
```

Comment:
`ResponseTooLarge` ERROR response message has its `ExtendedErrorData`, so I think here `libspdm_generate_error_response()` function should be changed to `libspdm_generate_extended_error_response()` function.

Reply:
Thanks. I have changed `libspdm_generate_error_response()` function should be changed to `libspdm_generate_extended_error_response()`.